manual/ec2-sru-20.2.txt

```diff
@@ -0,0 +1,222 @@
+#!/bin/sh
+
+set -e
+
+SRU_SERIES="bionic eoan focal xenial"
+
+# local values
+LP_USER=${LP_USER:-"lamoura"}
+
+# Provide USE_DEV_PPA=1 if we are SRU queue is still unapproved
+# This will use our ppa:cloud-init-dev/proposed
+if [ "${USE_DEV_PPA:-1}" -eq "0" ]; then
+    PROPOSED_SCRIPT=setup_proposed.sh
+else
+    PROPOSED_SCRIPT=setup_dev_proposed.sh
+fi
+
+
+# Manual EC2 upgrade and clean install validation
+cat > sethostname.yaml <<EOF
+## template: jinja
+#cloud-config
+ssh_import_id: [$LP_USER]
+hostname: SRU-worked-{{v1.cloud_name}}
+EOF
+
+cat > setup_proposed.sh <<EOF
+#/bin/bash
+mirror=http://archive.ubuntu.com/ubuntu
+echo deb \$mirror \$(lsb_release -sc)-proposed main | tee /etc/apt/sources.list.d/proposed.list
+apt-get update -q;
+apt-get install -qy cloud-init;
+EOF
+
+cat > setup_dev_proposed.sh <<EOF
+#/bin/bash
+add-apt-repository ppa:cloud-init-dev/proposed -y
+apt-get update -q;
+apt-get install -qy cloud-init;
+EOF
+
+# set -x outputs each commant to standard error, so an easy way to keen an eye
+# what's going on while saving a clean log file (stdout only) is using tee(1).
+set -x
+
+sshopts="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR"
+
+run_describe_command() {
+    dnsname=$1
+    query=$2
+    aws ec2 describe-instances --filter "Name=dns-name,Values=${dnsname}" --query $query | jq .[]
```

Comment:
Since you are dropping the quotes " in all call-sites, you could just use jq -r to return the raw output, then you don't need the tr -d elsewhere
```suggestion
    aws ec2 describe-instances --filter "Name=dns-name,Values=${dnsname}" --query $query | jq -r .[]
```